templates/base_index.html

```diff
@@ -135,44 +149,63 @@ <h3><a href="/desktop">Fast, secure and simple, Ubuntu powers millions of PCs wo
         }
       };
     </script>
+
     <script>
       if (window.localStorage && window.sessionStorage) {
-        var takeovers = document.getElementsByClassName('js-takeover');
-        for (t = 0; t < takeovers.length; ++t) {
-            // hide all takeovers
-            takeovers[t].classList.add('u-hide');
-        }
+        /**
+         * Choose a takeover
+         * ===
+         * 
+         * From the list of provided takeovers in the #takeovers template,
+         * choose one (that matches the client's language), and replace the
+         * base template with it.
+         */
+
+        var baseTakeover = document.querySelector('section#takeover');  // The base takeover element
```

Comment:
Why not use `getElementById`?

Reply:
Only because `querySelector` allows me to be more expressive to include the `section` element name. Although there's no need to be so opinionated.

I'm happy to change it to `getElementById` if you care at all.

Reply:
I've changed this.